lib/kernel/src/user_drv.erl

```diff
@@ -640,6 +642,8 @@ io_request({insert_chars, unicode, Chars}, TTY) ->
     write(prim_tty:handle_request(TTY, {insert, unicode:characters_to_binary(Chars)}));
 io_request({delete_chars, N}, TTY) ->
     write(prim_tty:handle_request(TTY, {delete, N}));
+io_request(clear, TTY) ->
```

Comment:
`clear` also needs to be handled in [`ssh_cli.erl`](https://github.com/erlang/otp/blob/master/lib/ssh/src/ssh_cli.erl#L412-L436).

Reply:
Should we expose `prim_tty:tgetstr/1` so we can use it from `ssh_cli`? Or should we expose a specific `prim_tty:clear/1` instead?

Reply:
I don't know how ssh is supposed to work in this regard. It seems like it is hard coded in the get_tty_command function right now, so I think we should be safe to do the same for `clear` in ssh.

ssh does some type of handshake procedure to get some of the information about the terminal that is connected, but how that is supposed to work I have no idea.

Reply:
I see. SSH actually assumes VT100 and "clear" is part of VT100:

> Cursor to Home
> ESC H 	 
>
> Move the cursor to the home position.

> Erase to End of Screen
> ESC J 	 
>
> Erase all characters from the active position to the end of the screen. The active position is not changed.

So I think we are good with hard-coding alongside the other characters?

Reply:
> So I think we are good with hard-coding alongside the other characters?

Sounds good